CONTRIBUTING.md

```diff
@@ -34,6 +34,27 @@ Some tips for hosting a great review club meeting:
   where to look in the codebase/GitHub/documentation, and let them do their
   own research from there.
 
+- When linking to code, use stable links that won't break later:
+
+  - for linking to code in the master branch, link to a recent commit hash in
+    the master branch of bitcoin/bitcoin, e.g.
+    [https://github.com/bitcoin/bitcoin/blob/23d3ae7a/src/addrman.cpp#L44](https://github.com/bitcoin/bitcoin/blob/23d3ae7a/src/addrman.cpp#L44).
+    You can use a short (8 character) commit id. REASON: linking directly to master
+    isn't stable because line numbers/files change over time as the HEAD commit of
+    master changes.
+  - for linking to code in the PR branch, ask one of the
+    bitcoin-core-review-club maintainers to add a tag of the current PR branch
+    to bitcoin-core-pr-reviews/bitcoin, and link to that branch or a commit on
```

Comment:
`s/bitcoin-core-pr-reviews/bitcoin-core-review-club/`

maybe add the link: [bitcoin-core-review-club/bitcoin](https://github.com/bitcoin-core-review-club/bitcoin) here and in line 55